011-InfraAsCode-ARM-DSC/Host/Solutions/armChallenges/challenge-12.parameters.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file is full of p20 references, I'll assume you know about all of these everywhere so I'll stop commenting on each.